src/js/game/LevelMVC/LevelModel.js

```diff
@@ -46,6 +46,15 @@ module.exports = class LevelModel {
     this.groundDecorationPlane = new LevelPlane(this.initialLevelData.groundDecorationPlane, this.planeWidth, this.planeHeight);
     this.shadingPlane = [];
     this.actionPlane = new LevelPlane(this.initialLevelData.actionPlane, this.planeWidth, this.planeHeight, true);
+
+    for (let i = 0; i < this.actionPlane.length; ++i) {
+      if (this.actionPlane[i].blockType.substring(0,12) === "redstoneWire") {
+        let y = Math.floor(i / this.planeHeight);
+        let x = i - (y * this.planeHeight);
+        this.determineRedstoneSprite(x, y, this.actionPlane[i]);
+      }
+    }
```

Comment:
Can we implement this in `LevelPlane` now that it exists?  I _think_ it makes more sense to have this functionality on the `actionPlane` object (so it updates automatically when `actionPlane.setBlockAt` is called) but not 100% sure.